mod/notifications/views/default/notifications/subscriptions/forminternals.php

```diff
@@ -6,40 +6,30 @@
  */
 
```

Comment:
Agreed on this one. Existing code is a total mess. We'll just have to test this manually very carefully.